x-pack/plugins/cases/docs/cases_client_typedoc.json

```diff
@@ -2,24 +2,26 @@
   "entryPoints": [
     "../server/client/client.ts",
     "../server/client/typedoc_interfaces.ts",
-    "../server/client/attachments",
+    "../server/client/attachments/client.ts",
     "../server/client/cases/client.ts",
     "../server/client/cases/get.ts",
     "../server/client/cases/push.ts",
     "../server/client/configure/client.ts",
+    "../server/client/metrics/client.ts",
     "../server/client/stats/client.ts",
-    "../server/client/sub_cases/client.ts",
     "../server/client/user_actions/client.ts"
   ],
+  "entryPointStrategy": "expand",
```

Comment:
I think this was the previous default behavior of `typedoc`: https://typedoc.org/guides/options/#entrypointstrategy